src/index.html

```diff
@@ -1,13 +1,76 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <div class="container">
+      <div
+        class="container__card card"
+        data-qa="card"
+        >
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__image card__image--position"
+        >
+
+        <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+
+        <p class="card__code">
+            Product code: 195434
+        </p>
+
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">
+            <p>Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__amount">
+          <p class="card__price">
+            Price:
+          </p>
+
+          <h1 class="card__money">
+            $2,199
+          </h1>
```

Comment:
point 12 in checklist **"Use semantic tags like h2, p ..."**  I think, semanticly it's not <h1>, just <p>